### ISSUE TITLE
Feature request: support for AWS tags on resources created by BatchForge

### ISSUE THREAD

**Issue by krokicki** [seqeralabs/nf-tower#306]
It would be useful if all resources created by Batch Forge (especially EC2 instances and FSx volumes) were tagged based on some custom key/value inputs in the Tower UI.

Our organization has a single AWS account, and we charge back usage costs to individual users based on a tagging scheme. Every resource in our AWS account is tagged with a project tag, which later makes it possible to use the AWS Cost Management feature to divvy up the costs among users each month. 

Batch Forge currently creates untagged resources which don't work with our chargeback scheme, so currently we can only use it on a very limited basis. Tag support in Tower would let us extend our current tagging scheme for Nextflow pipelines. Of course, I would also be interested to know if there is a better way to manage cost allocation that we're missing.

**Comment by pditommaso** [comment]
Yes, this is definitely on our roadmap, it should be expected by q3.
